src/styles/blocks/ingridients.scss

```diff
@@ -0,0 +1,104 @@
+.ingridients {
+  &__title {
+    @extend %text-h2;
+    font-size: 24px;
+    text-transform: uppercase;
+    grid-row: 1 / 2;
+    margin-bottom: 20px;
+
+    @include Desktop {
+      font-size: 36px;
+    }
+  }
+
+  &__link-container {
+    align-self: end;
+
+    @include Tablet {
+      grid-row: 2 / 3;
+    }
+  }
+
+  &__link {
+    @extend %links;
+
+    &:hover,
+    &:active {
+      padding-bottom: 10px;
+      border-bottom: 1px solid $secondary-color;
+    }
+  }
+
+  &__main {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    gap: 12px;
+    margin-right: -20px;
```

Comment:
Consider adding a comment to explain the negative margin value to improve code readability.